dashboard/app/models/levels/blockly.rb

```diff
@@ -609,4 +610,18 @@ def localized_shared_blocks(level_objects)
     end
     level_objects_copy
   end
+
+  def update_goal_override
+    if goal_override.present? && goal_override.is_a?(String)
+      self.goal_override = JSON.parse(goal_override)
+    end
+  end
+
+  def self.goal_override
```

Comment:
we aren't currently calling this method anywhere -- i think we want to call this for any Blockly level where `self.goal_override == nil` -- is that right? (making sure i understand before giving add'l guidance)

Reply:
I actually lifted this from StudioEc and now that I'm looking at it, I'm realizing that we don't want to call this - we only want to add a goalAnimation is the levelbuilder specifies one. Deleting . . .